.travis.yml

```diff
@@ -10,8 +10,8 @@ deploy:
   api_key:
 ​    secure: mU+yRvQsYADpzivO5NdShi/shZZtPpHwFA5vqCa63CDWTNCdQFsB2AGb6z/m/VFbY8XKhk/qYmDUtH9HyMW1LYu2m6vsA3XaeiVijw3Gf0nE0En8cfxUGwP95OoDi5VR1GMH5Nt3fzKqZbf4PM0NWEDKnS4rvles5rmGs1+gyIUKSeRqjXBiC+4yfJg/QrmCAYGAXZ8S1c2HzMJI+okiWiF8y6hYgykiXZRC0GNwNFpmI7YE9xp5BRyAYaU3Mk1JfiqAlX9/MIqdzzfT1yyE5zpqz4oenRoA8tefdsXYF2sCfPxN7YoF8QM4SUR6ItczNIEiU2pffOooHIf6+dstrylEUB0Nt2meR6k8vUYm3ExTZSyqWP+ud1SnKcjeSjNyiK1H2ARnHyZJRxtw6fRmvLRofB/YNufuOxR0HJ6LMyce5mnUBx7OZOxGhhnAeOYySNapJXcKPzE67gr7qWTz1XP1MEcWHF0UMsALQjkBEWXOWsv4mOXvzA3pn7XWMKdkeIYkiOt6+F/DufWJqOJ4n+TvVzqnjF9pwvqpHIbvO3ssQi28xczAj8PX5nV/t7b0pQLzMQ0QB8C8Q2t27qqZM8pWZ+PAqvHvFzG/QBA8kBW6WsfTq+UT4Gn7DX2ayBvQBXa3L9akNEbRUDm/oVcuQ2oEkGyQlvZLxsZ8I0hD7ww=
 
-on:
-  branch: main
+  on:
+    branch: main
 
 app: veg-share
```

Comment:
Apparently this might need to be adjusted too? According to that travis.com error

Reply:
perfect - let's merge this in and have that in mind for our next adjustment